docs/trackmate_scripting.rst

```diff
@@ -0,0 +1,126 @@
+Analyze OMERO timelapse images using the TrackMate API
+======================================================
+
+In this example we open an image stored in an OMERO server and use the `TrackMate <https://imagej.net/TrackMate>`_ API to analyze it.
+The core difference of this approach is that the generated track **can** be saved as
+OMERO ROIs.
+
+Description
+-----------
+
+First, we will show how to use the TrackMate API and the Scripting editor of Fiji.
+
+We will show:
+
+-  How to connect to OMERO using the JAVA API.
+
+-  How to retrieve an image.
+
+-  How to open the image using Bio-Formats.
+
+-  How to create a TrackMate model using its API.
+
+-  How to save the tracks as polygon ROIs in OMERO.
+
+
+Setup
+-----
+
+- Install Fiji on the local machine with the OMERO.insight-ij plugin. The installation instructions can be found at `here <https://omero-guides.readthedocs.io/en/latest/fiji/docs/installation.html>`_.
+
+Resources
+---------
+
+-  We will use a timelapse image available at `DV/iain/438CTR <https://downloads.openmicroscopy.org/images/DV/iain/438CTR/>`_
+
+-  Script: Groovy script for tracking timelapse images - :download:`tracking.groovy <../scripts/tracking.groovy>`.
+
+Step-by-Step
+------------
+
+In this section, we go through the steps required to analyze the data.
+The script used in this document is :download:`tracking.groovy <../scripts/tracking.groovy>`.
+
+A 'Tracking' Jupyter notebook is also available in the notebooks section.
+
+One advantage of the scripting approach is that we can save the generated tracks as 
+ROIs in an OMERO server.
+
+#. Launch Fiji.
+
+#. Open ``File > New > Script...``.
+
+#. Select ``Groovy`` as a language.
+
+#. Copy the content of :download:`tracking.groovy <../scripts/tracking.groovy>` in the text area.
+
+#. Enter the credentials to connect to the server and select an Image.
```

Comment:
```suggestion
#. A dialog will pop up. Enter the credentials to connect to the server and select an Image.
```

Reply:
But here is a mistake in the order of appearance I guess. That dialog can only appear after you clicked Run (next step below.)